### DISCUSSION TITLE
RelatedHubSites stopped working on refinement filters, I had to move it to Query template

### DISCUSSION THREAD

**Discussion by MrWorkingCat** [microsoft-search/pnp-modern-search#2022] [Q&A]
I was limiting results to items stored in SP sites associated to a hub by using the expression below in the **refinement filters** configuration:

`RelatedHubSites:equals("0db74199-bc20-45ad-a607-56a3222a422d")`

All was working fine in our pre-production prototype for 3 weeks.

Two days ago, I noticed the search results part was returning zero results, so I move the expression to the **Query template**, as per below, and got back the results.

`{searchTerms} RelatedHubSites:0db74199-bc20-45ad-a607-56a3222a422d`

I noticed this issue in my personal development environment and our company tenant. Likewise, moving the expression to the **Query template** worked in both environments.

**Questions:**

- Did I do something wrong the first time when used RelatedHubSites:equals("0db74199-bc20-45ad-a607-56a3222a422d") in refinement filters?

- Did something change in the SharePoint online backend?

- I am using contentclass:equals("STS_ListItem_DocumentLibrary") in the refinement filters in a second prototype search page and it is still working. Should I move it to the Query template?

**Comment by wobba** [comment]
Running equals using FQL is not a good idea. If you have nested hubs `RelatedHubSites` will contain a comma separated string. So you always want to do contains, not equals on this property.

**Reply by MrWorkingCat** [reply]
Thanks. Is contains a supported operand in FQL? 

I can't find it (or something like that) in the documentation below:

https://docs.microsoft.com/en-us/sharepoint/dev/general-development/fast-query-language-fql-syntax-reference#string

**Reply by wobba** [reply]
Change your fql to: `RelatedHubSites:string("0db74199-bc20-45ad-a607-56a3222a422d")` and it should work.
